tools/mob_suite/mob_recon.xml

```diff
@@ -1,16 +1,28 @@
-<tool id="mob_recon" name="MOB-Recon" version="1.4.1">
+<tool id="mob_recon" name="MOB-Recon" version="1.4.5">
   <description>Type contigs and extract plasmid sequences</description>
   <requirements>
-     <requirement type="package" version="1.4.1">mob_suite</requirement>
+     <requirement type="package" version="1.4.5">mob_suite</requirement>
   </requirements>   
   <command detect_errors="exit_code">
-    ln -s "${input}" "${input.name}";
-    mob_recon  --num_threads \${GALAXY_SLOTS:-4} --infile "${input.name}" --evalue '${adv_param.evalue}'  --unicycler_contigs '${adv_param.unicycler_contigs}' --run_circlator '${adv_param.run_circlator}' --run_typer --outdir '.';
+  <![CDATA[  
+    ln -s "${input}" "${input.name}"; 
+    
+    mob_recon  --num_threads \${GALAXY_SLOTS:-4} --infile "${input.name}" 
+    #if str($adv_param.unicycler_contigs) == "True":
+      --unicycler_contigs 
+    #end if   
+    #if str($adv_param.run_circlator) == "True":
+      --run_circlator 
+    #end if 
+    #if str($adv_param.min_length_condition.min_length_param) == "True":
+      --min_length ${adv_param.min_length_condition.min_length_value}
+    #end if 
+    '--run_typer' --outdir '.';
```

Comment:
Is there a reason why --run_typer is surrounded in single quoutes?

Reply:
It was just the first time I have seen that. If it works no need to remove them.